src/functionals/m0xy_fun.hpp

```diff
@@ -233,7 +234,7 @@ static num m05_c_anti(const parameter param_c[5],
 template <typename num>
 static num m05_c_para(const parameter param_c[5],
                       const num & chi2,
-                      const num & zet,
+                      const num & /* zet */,
```

Comment:
Should this be investigated?

Reply:
The value is not used, but it was easier to do this than change all the different places where it's called. It might be that it can be safely removed, but I cannot be sure.